Flamedriver.asm

```diff
@@ -1236,7 +1236,6 @@ zDoFMVolEnv:
 		push	bc							; Save bc
 		jr	nc, .skip_reg					; Branch if c bit shifted was zero
 		add	a, (hl)							; Add TL value to volume envelope
-		and	7Fh								; Strip sign bit
```

Comment:
This should actually do the same thing as above to prevent clipping. It can probably be simpler, because the volume envelopes are always positive.